### DISCUSSION TITLE
1.0.6 업데이트에서 카메라 위치 변경

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#40] [5. Q&A]
1.0.4의 카메라 높이는 1이었는데 1.0.6 에서는 1.97정도로 수정된것으로 보입니다...
앞으로는 카메라 높이가 변경된것으로 가는건가요?
매 업데이트마다 카메라 높이가 변경되는건가요?

**Comment by moondory77-naverz** [comment]
안녕하세요. 금번 1.0.6 업데이트에서,
각 캐릭터의 체형이 달라지는 다양한 경우를 지원하기 위해, 
제페토 카메라의 디폴트 position값 자체가 (비추는 캐릭터의) 머리를 기준으로 조정되도록 업데이트 되었습니다. 

또한 카메라의 position은 ZepetoPlayers 컴포넌트의 LookOffset 값을 설정함으로써
디폴트 position에 대해, 추가적인 위치조정이 가능함을 참고 부탁드리겠습니다.

![modify look offset](https://user-images.githubusercontent.com/93751484/145151796-1c3f421a-fa35-43d6-a6f6-ec7ec86458f5.png)

다양한 체형에 대한 케이스를 지원하기 위해 추가된 업데이트이며
매 업데이트마다 카메라의 디폴트 position가 변경되지는 않습니다. 양해 부탁드리겠습니다.

늘 감사합니다. :)
문의사항이 있으실 경우, 언제든 부탁드리겠습니다.
